hack/ci-e2e-kind-gardenadm.sh

```diff
@@ -14,11 +14,13 @@ clamp_mss_to_pmtu
 
 ensure_glgc_resolves_to_localhost
 
-make kind-up
+make kind-single-node-up
 
+# export all container logs and events after test execution
 trap "
-  ( export_artifacts "gardener-local" )
-  ( make kind-down )
+  ( export KUBECONFIG=$PWD/example/gardener-local/kind/multi-zone/kubeconfig; export_artifacts 'gardener-operator-local'; export_resource_yamls_for garden)
+  ( export KUBECONFIG=$PWD/dev-setup/kubeconfigs/virtual-garden/kubeconfig; export cluster_name='virtual-garden'; export_resource_yamls_for seeds shoots; export_events_for_shoots)
+  ( make kind-multi-zone-down )
```

Comment:
Shouldn't this be, since we are doing `make kind-single-node-up`
```suggestion
  ( make kind-single-node-down )
```